usb/esp_tinyusb/include/tusb_config.h

```diff
@@ -114,6 +118,17 @@ extern "C" {
 // MSC Buffer size of Device Mass storage
 #define CFG_TUD_MSC_BUFSIZE         CONFIG_TINYUSB_MSC_BUFSIZE
 
+// AUDIO macros
+#define CFG_TUD_AUDIO_FUNC_1_DESC_LEN                                 TUD_AUDIO_MIC_ONE_CH_DESC_LEN
```

Comment:
@tore-espressif , 
When you will have time, could you please check the configuration parameters for the audio device here. 
I think couple of things: 
- What do we really need to configure? 
- Is there any better place to define such variables in the `esp_tinyusb` component?